src/config.rs

```diff
@@ -0,0 +1,58 @@
+use log::error;
+use std::{
+    fs::{self, create_dir},
+    path::Path,
+};
+
+use crate::keyboard;
+
+pub fn get_config_root() -> String {
+    let config_path: String;
+
+    if let Ok(home) = std::env::var("HOME") {
+        config_path = Path::new(&home)
+            .join(".config/wlxroverlay")
+            .to_str()
+            .unwrap()
+            .to_string();
+    } else {
+        config_path = "/tmp/wlxroverlay".to_string();
+        error!("Err: $HOME is not set, using {}", config_path);
+    }
+    // Make sure config directory is created
+    let _ = create_dir(&config_path);
+    config_path
+}
```

Comment:
This doesn't support XDG_CONFIG_HOME. Maybe we should just use a crate like [`xdg`](https://crates.io/crates/xdg)